modules/utility/binds/which-key/which-key.nix

```diff
@@ -1,7 +1,77 @@
-{lib, ...}: let
-  inherit (lib) mkEnableOption;
+{
+  config,
+  lib,
+  ...
+}: let
+  inherit (lib) mkEnableOption mkOption mkOptionType types;
```

Comment:
Same as my previous comment. See https://noogle.dev for parent attributes of each lib call.